Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -822,8 +824,9 @@ void Scene_c3t3_item::drawEdges(CGAL::Three::Viewer_interface* viewer) const {
     ncthis->d->initializeBuffers(viewer);
   }
 
-  if(renderingMode() == Wireframe)
+  if(renderingMode() == Wireframe && d->is_grid_shown)
   {
+
```

Comment:
To minimize the patch, remove this empty line that was not in the code before the changes.